sources/web/datalab/polymer/test/github-cache-test.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+describe('GithubCache', () => {
+  const fixture = new GithubCache();
```

Comment:
This isn't really a fixture; its state is shared across these tests right?
Maybe we should just create a new `GithubCache` object inside each test.

Reply:
Done.